Library/Homebrew/utils/github.rb

```diff
@@ -296,7 +296,11 @@ def search_code(**qualifiers)
   def issues_for_formula(name, options = {})
     tap = options[:tap] || CoreTap.instance
     tap_full_name = options[:tap_full_name] || tap.full_name
-    search_issues(name, state: "open", repo: tap_full_name, in: "title")
+    if options.key?(:state)
+      search_issues(name, state: options[:state], repo: tap_full_name, in: "title")
```

Comment:
```suggestion
    if state.present?
      search_issues(name, state: state, repo: tap_full_name, in: "title")
```

Reply:
Although I wonder what happens if `state: nil` is passed through? Maybe `state` can be passed unconditionally?